api.go

```diff
@@ -70,33 +70,40 @@ type ClientPersistent struct {
 	client        *Client
 }
 
-// GetVersion returns version
-func (c *Client) GetVersion() (string, error) {
-	resp, err := c.api.DoAndGetResponseBody(
+func (c *Client) GetVersion() (ver string, err error) {
+	var resp *http.Response
+	fmt.Println(c.GetToken())
```

Comment:
Please delete this fmt statement from here, this was added for debugging purpose.